examples/reconfigurationBeam/circularReconfiguration.m

```diff
@@ -0,0 +1,224 @@
+mesh.conecCell = { } ;
+% then the first welded node is defined with material (M) zero since nodes don't have material, the first element (E) type (the first entry of the `elements` struct), and (B) is the first entry of the the `boundaryConds` struct. For (I) no non-homogeneous initial condition is considered (then zero is used) and finally the node is assigned:
+mesh.conecCell{ 1, 1 } = [ 0 1 1 0  1] ;
+% Next the frame elements MEBI parameters are set. The frame material is the first material of `materials` struct, then $1$ is assigned. The second entry of the `elements` struct correspond to the frame element employed, so $2$ is set. Finally no BC and no IC is required for this element, then $0$ is used.  Consecutive nodes build the element so then the `mesh.conecCell` is:
+for i=1:numElements,
+  mesh.conecCell{ i+1,1 } = [ 1 2 0 0  i i+1 ] ;
+end
+%md 
+%md## Run ONSAS
+%md---------------------
+%md Declare a global variable to store drag 
+%
+global globalFDrag
+global globalNIter
+globalFDrag = zeros(analysisSettings.finalTime, 1) ;
+globalNIter = zeros(analysisSettings.finalTime + 1, 1) ;
+%
+% Run ONSAS 
+%
+[matUs] = ONSAS( materials, elements, boundaryConds, initialConds, mesh, analysisSettings, otherParams ) ;
+%
+%### Numeric solution
+%
+% The numerical solution is extracted. First the reference coordinaes
+xref = mesh.nodesCoords(:,1) ;
+yref = mesh.nodesCoords(:,2) ;
+zref = mesh.nodesCoords(:,3) ;
+% 
+%## Validation results
+%---------------------
+% Compute the values of R and CyCd:
+numLoadSteps = size(matUs, 2) ;
+timeVec = linspace(0,analysisSettings.finalTime, numLoadSteps) ;
+% initialize vectors
+Cy = zeros(numLoadSteps-1, 1)                 ;
+R  = zeros(numLoadSteps-1, 1)                 ;
+C_d = feval( elements(2).aeroCoefs{1}, 0 , 0) ;
+% fill them
+for windVelStep = 1:numLoadSteps - 1
+    % Compute dimensionless magnitudes 
+    windVel         = feval( analysisSettings.fluidProps{3,:}, 0, timeVec(windVelStep + 1 ) ) ;
+    normWindVel     = norm( windVel )                                                         ;
+    dirWindVel      = windVel / normWindVel                                                   ;
+    Cy(windVelStep) =  1/2 * rhoF * normWindVel^2 * (l)^3 *d / (E*Izz)                        ;
+
+    % numeric drag 
+    FDragi = globalFDrag(windVelStep)                 ;
+    FDRef  = 1/2 * rhoF * normWindVel^2 * C_d * d * l ;
+    R(windVelStep) =  abs(FDragi)/(FDRef )            ;
+
+end
+%
+%### Gosselin et.Al 2010 solution
+%
+% resudrag (cycd, R) and def wich contains de deformed configuration for 10^i cycyd values: 
+disp('pwd is ')
+pwd
+disp('path data is ')
+path_data = strcat('/github/workspace/examples/reconfigurationBeam/','/Gosselin2010_data.mat' )
+% path_data = strcat('.','/Gosselin2010_data.mat' )
+load(path_data, 'def', 'resudrag')
```

Comment:
if https://github.com/ONSAS/ONSAS.m/pull/557/files#r970141594 is done then this (i think) should be only
```suggestion
load( 'Gosselin2010_data.mat', 'def', 'resudrag')
```